### DISCUSSION TITLE
Nats implementation

### DISCUSSION THREAD

**Discussion by gopinathjcs** [apalis-dev/apalis#375] [Ideas]
Hi, I planning to implement NATS and thought of contributing it back to the repo.  What is the plan of AMQP repo, is it going to be in separate repo or combined with the main apalis repo.  Also when is 0.6 released planned.

Regards
Gopi

**Comment by geofmureithi** [comment]
I already have a wip for jetstream, it will be part of 0.6 which should be out before the end of the month. There some bugs and some extra polishing up needed.

Regarding `amqp`, I plan to bring everything together in the 0.6 release. They wont be in the same repo, but it will be updated appropriately.

Am open to ideas though. Why do you need it in the apalis repo?

**Comment by gopinathjcs** [comment]
Thanks for the update. Just wanted to check where to contribute.

I have requirement to have both NATS and postgres based in Single worker  also I was thinking on Request/Response pattern of NATS.

One more clarification. Is there way I can have multiple Task registered in same worker?  I'm planning to build workflow engine on top of Apalis, any thoughts on this?

**Reply by geofmureithi** [reply]
https://github.com/geofmureithi/apalis/issues/376

**Comment by gopinathjcs** [comment]
Hi, I would like to connect with in person, how can I reach you? any DM?.

**Reply by geofmureithi** [reply]
Sure you could reach out. Do you have a specific issue in mind? (Usually I prefer to handle issues via GitHub so that the contents are available for other users)
I can provide an email or telegram if you need to have a private consultation.

**Reply by gopinathjcs** [reply]
Its more for a work opportunity/private Consultation in line with what you have built.Pls share you email and we can setup sometime to chat.
